sentry-core/src/backtrace_support.rs

```diff
@@ -53,19 +54,15 @@ static ref WELL_KNOWN_BORDER_FRAMES: Vec<&'static str> = {
     #[cfg(feature = "with_log")] {
         rv.push("<sentry::integrations::log::Logger as log::Log>::log");
     }
-    #[cfg(feature = "with_error_chain")] {
-        rv.push("error_chain::make_backtrace");
-    }
     rv
 };
 
 static ref SECONDARY_BORDER_FRAMES: Vec<(&'static str, &'static str)> = {
     #![allow(unused_mut)]
     let mut rv = Vec::new();
-    #[cfg(feature = "with_error_chain")] {
-        rv.push(("error_chain::make_backtrace", "<T as core::convert::Into<U>>::into"));
-    }
-    {rv}
+    // TODO: figure out how to expose these secondary frames to be configured
```

Comment:
We could do this by adding an event processor that scans for the frame and then removes it. Basically make the internal logic reusable and move it to a processor instead of having it in the client.

Reply:
Hm… maybe have a function which optionally trims just one match, then an integration can easily just chain two calls together. that would make things quite easy I think… Lets do that as a followup ;-)